src/Symfony/Bundle/Resources/config/json_schema.xml

```diff
@@ -22,6 +22,7 @@
             <argument type="service" id="api_platform.metadata.property.metadata_factory" />
             <argument type="service" id="api_platform.name_converter" on-invalid="ignore" />
             <argument type="service" id="api_platform.resource_class_resolver" />
+            <argument type="collection" />
```

Comment:
we can safely inject the parameter here + on-invalid="ignore", it defaults to an empty array anyways in the constructor

Reply:
Done. It is a much better idea!